jstz_cli/src/accounts/account.rs

```diff
@@ -0,0 +1,64 @@
+use anyhow::Result;
+use jstz_crypto::{hash::Blake2b, public_key::PublicKey, public_key_hash::PublicKeyHash};
+use serde::{Deserialize, Serialize};
+use tezos_crypto_rs::bls::keypair_from_ikm;
+
+fn create_address(pk: PublicKey) -> String {
+    PublicKeyHash::try_from(&pk).unwrap().to_base58()
+}
+
+// Represents an individual account
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct Account {
+    alias: String,
+    address: String,
+    secret_key: String,
+    public_key: String,
+}
+
+impl Account {
+    pub fn new(
+        alias: String,
+        address: String,
+        secret_key: String,
+        public_key: String,
+    ) -> Self {
+        Account {
+            alias,
+            address,
+            secret_key,
+            public_key,
+        }
+    }
```

Comment:
`new` should only be implemented if you:
1. Want to keep the fields private or,
2. Non-trivial logic for initializing the struct value
Otherwise the idiomatic way is to use the Rust struct constructor e.g. `Account { ... }`. 

Given this, I think this `new` function should be removed